### ISSUE TITLE
Missed msdia120.dll

### ISSUE THREAD

**Issue by AndreyAkinshin** [PerfDotNet/BenchmarkDotNet#62]
In `BenchmarkDotNet.IntegrationTests.csproj`, we have the following lines:
```
  <PropertyGroup>
    <PostBuildEvent>xcopy /F /Y $(ProjectDir)..\CLRMD\x86\msdia120.dll $(TargetDir)</PostBuildEvent>
  </PropertyGroup>
```
But we don't have the `x86\msdia120.dll` file in `CLRMD`, building fails.

**Comment by mattwarren** [comment]
whoops, I messed that up (definitely a case of "*it only works on my machine!!*")

I'll get it sorted

**Comment by mattwarren** [comment]
Fixed in a3e2dc8
